spec/features/saml/loa3_sso_spec.rb

```diff
@@ -7,11 +7,12 @@
   context 'First time registration' do
     it 'redirects to original SAML Authn Request after IdV is complete' do
       allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
-      saml_authn_request = auth_request.create(loa3_with_bundle_saml_settings)
-      visit saml_authn_request
-
+      settings = saml_settings
+      settings.authn_context = Saml::Idp::Constants::LOA3_AUTHN_CONTEXT_CLASSREF
+      saml_authn_request = auth_request.create(settings)
```

Comment:
huh it worked for me too. Problems must have been with an old implementation or something. Fixed at 41a53f1